src/core/api.ts

```diff
@@ -2022,7 +2020,7 @@ export class Api<R extends RawApi = RawApi> {
      * @param payload Bot-defined invoice payload, 1-128 bytes. This will not be displayed to the user, use for your internal processes.
      * @param provider_token Payments provider token, obtained via BotFather
      * @param currency Three-letter ISO 4217 currency code, see more on currencies
-     * @param prices Price breakdown, a list of components (e.g. product price, tax, discount, delivery cost, delivery tax, bonus, etc.)
+     * @param prices Price breakdown, a list of components (e.g. product price and name), max 15 objects.
```

Comment:
I don't know where from you're taking this. https://core.telegram.org/bots/api#sendinvoice says that the current version is correct.